index.html

```diff
@@ -37,7 +37,7 @@
         function evalLinks() {
             if (typeof vis === 'undefined') {
                 console.log(document.getElementById("linkStatus"));
-                document.getElementById("linkStatus").innerHTML = "Note: The latest version (4.20.0) is not yet available on cdnjs, <a href='https://cdnjs.com/libraries/vis'>click here</a> to to pick the latest available version.<br />";
+                document.getElementById("linkStatus").innerHTML = "Note: The latest version (4.20.1) is not yet available on cdnjs, <a href='https://cdnjs.com/libraries/vis'>click here</a> to to pick the latest available version.<br />";
```

Comment:
This check is not accurate because 4.20.0 is still loaded on the page. Need to update the version loaded on the page to 4.20.1 as well in order to correctly detect if new Vis version is available on cdnjs. (Spoiler: It already is, but it's still worth getting this check right)